docs/development.rst

```diff
@@ -31,5 +31,4 @@ setuptools changes.  You have been warned.
    :maxdepth: 1
 
    developer-guide
-   formats
```

Comment:
This one was renamed to `python_eggs` and is currently linked from `deprecated/index` TOC.